[ID] Update `BN General Information` (#5533)
ppy/osu-wiki: wiki/People/The_Team/Beatmap_Nominators/General_Information/id.md

# Informasi umum untuk Beatmap Nominator

> Apakah kamu seorang [Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators) *(BN)*, atau tertarik menjadi salah satunya? jika ya, kamu datang ke tempat yang tepat! Artikel ini berisi semua informasi umum yang perlu kamu ketahui sebagai seorang Beatmap Nominator baru.

Reviewer: ```suggestion
Apakah kamu seorang [Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators) *(BN)*, atau tertarik menjadi salah satunya? Jika ya, kamu datang ke tempat yang tepat! Artikel ini berisi semua informasi umum yang perlu kamu ketahui sebagai seorang Beatmap Nominator baru.
```

Reviewer: better capitalized

## Tautan penting

> - [Peraturan Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators/Rules): Garis besar bagaimana cara melakukan berbagai tindakan, serta bagaimana cara untuk tidak melakukan sesuatu yang dapat melanggar ketentuan-ketentuan yang telah diberikan dalam situasi apa pun, jadi pastikan untuk membacanya dengan cermat! kamu selalu dapat bertanya kepada seorang anggota dari [Nomination Assessment Team](The_Team/Nomination_Assessment_Team) *(NAT)* jika kamu tidak yakin tentang sesuatu, dan mereka akan membantu kamu dengan senang hati.

Reviewer: ```suggestion
- [Peraturan Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators/Rules): Garis besar cara untuk melakukan berbagai tindakan, serta bagaimana cara untuk tidak melakukan sesuatu yang dapat melanggar ketentuan-ketentuan yang telah diberikan dalam situasi apa pun, jadi pastikan untuk membacanya dengan cermat! Kamu selalu dapat bertanya kepada seorang anggota dari [Nomination Assessment Team](The_Team/Nomination_Assessment_Team) *(NAT)* jika kamu tidak yakin tentang sesuatu, dan mereka akan membantu kamu dengan senang hati.
```

> - [Peraturan Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators/Rules): Garis besar bagaimana cara melakukan berbagai tindakan, serta bagaimana cara untuk tidak melakukan sesuatu yang dapat melanggar ketentuan-ketentuan yang telah diberikan dalam situasi apa pun, jadi pastikan untuk membacanya dengan cermat! kamu selalu dapat bertanya kepada seorang anggota dari [Nomination Assessment Team](The_Team/Nomination_Assessment_Team) *(NAT)* jika kamu tidak yakin tentang sesuatu, dan mereka akan membantu kamu dengan senang hati.

Reviewer 2: ```suggestion
- [Peraturan Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators/Rules): Garis besar bagaimana cara melakukan berbagai tindakan, serta bagaimana cara untuk tidak melakukan sesuatu yang dapat melanggar ketentuan-ketentuan yang telah diberikan dalam situasi apa pun, jadi pastikan untuk membacanya dengan cermat! kamu selalu dapat bertanya kepada seorang anggota dari [Nomination Assessment Team](/wiki/The_Team/Nomination_Assessment_Team) *(NAT)* jika kamu tidak yakin tentang sesuatu, dan mereka akan membantu kamu dengan senang hati.
```

> - [Kode Etik untuk Mapping dan Modding](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping): Sebagai seorang BN, kamu diharapkan dapat menjadi sosok yang patut dicontoh di dalam komunitas, menjadi seorang yang orang lain kagumi dan hormati, jadi pelanggaran dapat membuat kamu ditegur atau bahkan dikeluarkan.

Reviewer: ```suggestion
- [Kode Etik untuk Mapping dan Modding](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping): Sebagai seorang BN, kamu diharapkan dapat menjadi sosok yang patut dicontoh di dalam komunitas, menjadi seorang yang orang lain kagumi dan hormati, jadi perbuatan yang melanggar pengaturan dapat membuat kamu ditegur atau bahkan dikeluarkan.
```

> - [Ranking Criteria](/wiki/Ranking_Criteria): Jika kamu sudah menjadi seorang BN, kamu harus terbiasa dengan ini. Pada dasarnya, ini berfungsi untuk menjelaskan aspek mapping yang diperbolehkan dan tidak diperbolehkan di bagian Ranked, serta beberapa pedoman umum yang harus diikuti dalam kebanyakan kasus.

Reviewer: ```suggestion
- [Ranking Criteria](/wiki/Ranking_Criteria): Jika kamu sudah menjadi seorang BN, kamu seharusnya sudah terbiasa dengan ini. Pada dasarnya, hal ini berfungsi untuk menjelaskan aspek mapping yang diperbolehkan dan tidak diperbolehkan di bagian Ranked, serta beberapa pedoman umum yang harus diikuti dalam kebanyakan kasus.
```

> - [Informasi Beatmap Veto](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto): Untuk beatmap dengan masalah kualitas secara objektif yang sebetulnya tidak melanggar aturan dan pedoman dari Ranking Criteria, tetapi yang menurut kamu membuat beatmap tersebut tidak sesuai untuk bagian Ranked, beatmap veto memungkinkan kamu menahan beatmap dari Kualifikasi.

Reviewer: ```suggestion
- [Informasi Beatmap Veto](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto): Untuk beatmap dengan masalah kualitas secara objektif yang sebetulnya tidak melanggar aturan dan pedoman dari Ranking Criteria, tetapi yang menurut kamu membuat beatmap tersebut tidak sesuai untuk bagian Ranked, beatmap veto memungkinkan kamu untuk menahan beatmap dari bagian Qualification.
```

- [Mapset Verifier](https://osu.ppy.sh/community/forums/topics/943895): Sebuah alat yang dibuat oleh anggota NAT yaitu [Naxess](https://osu.ppy.sh/users/8129817). Alat ini memeriksa berbagai masalah unrankable, serta memiliki fitur snapshot agar lebih mudah dalam membandingkan versi map yang bermanfaat saat pengecekan ulang. **Ini tidak menggantikan pemeriksaan beatmap sendiri**, tetapi digunakan sebagai alat untuk membantu memeriksa ulang sehingga tidak ada yang terlewat.

## Kemampuan Beatmap Nominator

Nominasi dan mereset nominasi adalah bukti dari usaha seorang BN saat memeriksa map, menunjukkan kemajuan sebuah map melalui proses ranking. Sebagai seorang BN kamu memiliki akses ke alat-alat ini dan harus menggunakannya dengan bijak.

### Nominasi

![tombol nominasi](img/nominate_id.png)

> **Tombol ini memungkinkan kamu untuk menominasi beatmap.** Untuk menominasi sebuah beatmap, mapset harus memiliki setidaknya 5 Hype dan tidak ada tanda Masalah atau Saran yang saat ini belum terselesaikan. Pastikan kamu puas dengan semua tingkat kesulitan, bahkan mode permainan lainnya, sebelum menempatkan ini. Jika beatmap sebelumnya diveto, kamu harus memastikan veto tersebut sudah diselesaikan sebelum memajukan mapset. Ketika sebuah beatmap memiliki 2 nominasi, itu akan dikualifikasi dan masuk ke antrian ranking.

Reviewer 2: ```suggestion
**Tombol ini memungkinkan kamu untuk menominasi beatmap.** Untuk menominasi sebuah sebuah beatmap, mapset harus memiliki setidaknya 5 Hype dan tidak ada tanda Masalah atau Saran yang saat ini belum terselesaikan. Sebelum memberikan nominasi, pastikan kamu puas dengan semua tingkat kesulitan, bahkan untuk mode permainan lainnya. Jika beatmap sebelumnya pernah diveto, kamu harus memastikan veto tersebut sudah diselesaikan sebelum meneruskan mapset. Ketika sebuah beatmap memiliki 2 nominasi, beatmap tersebut akan menuju ke bagian kualifikasi dan masuk ke antrian ranking.
```

Reviewer 2: this probably wayyyy better

### Diskualifikasi atau mereset nominasi

> ![tombol diskualifikasi dan reset](img/reset_dq_id.png)

Reviewer 2: ```suggestion
![tombol diskualifikasi dan tombol reset](img/reset_dq_id.png)
```

Reviewer 2: could it get the own word since it's doesn't have to attach with `diskualifikasi `

> **Tombol ini memungkinkan kamu untuk menominasi beatmap.** Untuk menominasi sebuah sebuah beatmap, mapset harus memiliki setidaknya 5 Hype dan tidak ada tanda Masalah atau Saran yang saat ini belum terselesaikan. Sebelum memberikan nominasi, pastikan kamu puas dengan semua tingkat kesulitan, bahkan untuk mode permainan lainnya. Jika beatmap sebelumnya pernah diveto, kamu harus memastikan veto tersebut sudah diselesaikan sebelum meneruskan mapset. Ketika sebuah beatmap memiliki 2 nominasi, beatmap tersebut akan menuju ke bagian kualifikasi dan masuk ke antrian ranking.

Reviewer: ```suggestion
**Tombol ini memungkinkan kamu untuk menominasi beatmap.** Untuk menominasi sebuah beatmap, mapset harus memiliki setidaknya 5 Hype dan tidak ada tanda Masalah atau Saran yang saat ini belum terselesaikan. Sebelum memberikan nominasi, pastikan kamu puas dengan semua tingkat kesulitan, bahkan untuk mode permainan lainnya. Jika beatmap sebelumnya pernah diveto, kamu harus memastikan veto tersebut sudah diselesaikan sebelum meneruskan mapset tersebut. Ketika sebuah beatmap memiliki 2 nominasi, beatmap tersebut akan menuju ke bagian kualifikasi dan masuk ke antrian ranking.
```

**Ini memungkinkan kamu untuk mereset perkembangan dari sebuah beatmap dalam proses ranking.** Jika sebuah beatmap saat ini dinominasi atau dikualifikasi, kamu sebagai seorang nominator diberikan kemampuan untuk mereset nominasi pada beatmap dengan memposting sebuah isu menggunakan tanda Masalah. Jika sebuah beatmap dikualifikasi, itu hanya dapat didiskualifikasi oleh seorang full BN. Opsi ini biasanya digunakan ketika ada masalah besar atau masalah unrankable dalam beatmap yang harus ditangani sebelum mempertimbangkannya untuk ranking.

> Langkah-langkah untuk Mendiskualifikasi Beatmap:

Reviewer 2: ```suggestion
Langkah-langkah untuk mendiskualifikasi beatmap:
```

Reviewer 2: should be lowercase

> **Ini memungkinkan kamu untuk mereset perkembangan dari sebuah beatmap dalam proses ranking.** Jika sebuah beatmap saat ini dinominasi atau dikualifikasi, kamu sebagai seorang nominator diberikan kemampuan untuk mereset nominasi pada beatmap dengan memposting sebuah isu menggunakan tanda Masalah. Jika sebuah beatmap dikualifikasi, itu hanya dapat didiskualifikasi oleh seorang full BN. Opsi ini biasanya digunakan ketika ada masalah besar atau masalah unrankable dalam beatmap yang harus ditangani sebelum mempertimbangkannya untuk ranking.

Reviewer: ```suggestion
**Ini memungkinkan kamu untuk mereset perkembangan dari sebuah beatmap dalam proses ranking.** Jika sebuah beatmap saat ini dinominasi atau dikualifikasi, kamu sebagai seorang nominator diberikan kemampuan untuk mereset nominasi pada beatmap dengan memposting sebuah isu menggunakan tanda Masalah. Jika sebuah beatmap dikualifikasi, beatmap itu hanya dapat didiskualifikasi oleh seorang full BN. Opsi ini biasanya digunakan ketika ada masalah besar atau masalah unrankable dalam beatmap yang harus ditangani sebelum mempertimbangkannya untuk masuk ke bagian ranking.
```

- Pastikan bahwa kamu berada di diskusi beatmap pada set yang benar.
- Tulis alasan diskualifikasi kamu ke dalam kolom teks. Pastikan bahwa baris pertama merangkum diskualifikasi karena inilah yang ditampilkan di bagian atas halaman diskusi.
- Jika diskualifikasi ditautkan ke sebuah laporan, tautkan juga postingan diskusi beatmap terkait.
- Tekan tombol **Diskualifikasi** di bawah kolom teks (salah satu yang menggantikan tombol "Masalah").

> Sebagai perpanjangan dari kemampuan ini, kamu juga bisa [memveto sebuah beatmap](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto), mencegah sebuah beatmap dinominasi kembali sampai kamu memberikan izin atau veto dicabut oleh mediasi veto. Namun, mengajukan veto juga mengharuskan kamu mendiskusikan masalah ini dengan mapper dengan benar dan mencoba mencapai mufakat. Membahas masalah dengan Nominator asli akan bermanfaat dan dapat memberi kamu perspektif baru tentang masalah tersebut, tetapi kamu tidak diwajibkan untuk melakukannya.

Reviewer: ```suggestion
Sebagai tambahan dari kemampuan ini, kamu juga bisa [memveto sebuah beatmap](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto), mencegah sebuah beatmap dinominasi kembali sampai kamu memberikan izin atau veto dicabut oleh mediasi veto. Namun, mengajukan veto juga mengharuskan kamu mendiskusikan masalah ini dengan mapper dengan benar dan mencoba mencapai mufakat. Membahas masalah dengan Nominator yang awal akan bermanfaat dan dapat memberi kamu perspektif baru tentang masalah tersebut, tetapi kamu tidak diwajibkan untuk melakukannya.
```

## Fungsi Beatmap Nominator tambahan

Selain dapat menominasi beatmap atau mereset nominasinya, BN juga memiliki fungsi lain.

### Mediasi veto

Saat mediasi veto dimulai, seorang BN akan dipilih menjadi juri secara acak. Jika kamu adalah salah satu anggota terpilih untuk mediasi veto, NAT akan mengirimkan sebuah pesan dengan tambahan informasi tentang cara memberikan veto menggunakan [situs BN](http://bn.mappersguild.com). Jika kamu tidak dapat berpartisipasi, beritahu kepada seorang anggota NAT sehingga BN lain dapat dipilih sebagai juri pengganti.

### Quality Assurance

> Full BN dapat melakukan Quality Assurance dengan mengecek beatmap-beatmap yang telah dikualifikasi kapan pun. Papan Quality Assurance dapat dilihat pada [situs BN](https://bn.mappersguild.com/qualityassurance).

Reviewer 2: ```suggestion
Full BN dapat melakukan Quality Assurance dengan mengecek masalah pada beatmap-beatmap yang telah dikualifikasi kapan pun. Papan Quality Assurance dapat dilihat pada [situs BN](https://bn.mappersguild.com/qualityassurance).
```

Reviewer 2: you forgot that one

### Peninjauan Konten

> Semua BN dapat berkontribusi dalam memoderasi lagu-lagu dan gambar-gambar dengan mengirimkan konten untuk ditinjau atau memberikan suara pada konten tersebut. Proses pemungutan suara secara utuh dan bagaimana suara-suara tersebut dihitung dapat dibaca di [artikel proses pengkajian konten](/wiki/Rules/Content_Voting_Process).

Reviewer 2: ```suggestion
Semua BN dapat berkontribusi ke dalam moderasi lagu dan gambar dengan mengirimkan konten untuk ditinjau atau diberikan suara pada konten tersebut. Proses pemungutan suara secara utuh dan bagaimana suara-suara tersebut dihitung dapat dibaca di [artikel proses pengkajian konten](/wiki/Rules/Content_Voting_Process).
```
